examples/kubernetes/scripts/05-2-run-inside-vms-kubernetes-worker.sh

```diff
@@ -65,7 +65,7 @@ After=docker.service
 Requires=docker.service
 
 [Service]
-ExecPre=/bin/mount bpffs /sys/fs/bpf -t bpf
+#ExecStartPre=/bin/mount bpffs /sys/fs/bpf -t bpf
```

Comment:
Maybe leave a comment that this needs to be uncommented if the script is used in the non vagrant environment where the bpffs is not already mounted.